assets/src/domain/eventEditor/ui/tickets/ticketsList/editable/EditableName.tsx

```diff
@@ -24,16 +24,13 @@ const EditableName: React.FC<EditableNameProps> = ({ className, entity: ticket,
 	);
 
 	const ticketName = ticket.name ? ticket.name : __('Edit title...');
+	const editableProps = {
+		className,
+		onSubmit,
+		defaultValue: ticketName,
+	};
 
-	return view === 'table' ? (
-		<InlineEditText className={className} ellipsis={false} onChange={onChangeName}>
-			{ticketName}
-		</InlineEditText>
-	) : (
-		<InlineEditHeading level={3} className={className} onChange={onChangeName}>
-			{ticketName}
-		</InlineEditHeading>
-	);
+	return view === 'table' ? <InlineEditText {...editableProps} /> : <InlineEditHeading {...editableProps} />;
```

Comment:
same deal as the date name:

```suggestion
	return view === 'table' ? <InlineEditText {...editableProps} /> : <h3><InlineEditHeading {...editableProps} /></h3>;
```